test/fixtures/get-orderbook-response.json

```diff
@@ -169,6 +169,26 @@
         "direction": "buy",
         "quantity": {
           "currency": "USD",
+          "value": "0.003967400879423823",
```

Comment:
This looks a bit off somehow. I'm supposed to read quantity as the amount the person wants to buy? Here it's "want to buy/get 0.00397 USD and pay 1.242 BTC ? It seems like the issue is the wrong way around somehow ??

(Checking now)